feat(vercel): Use Sharp in dev instead of Squoosh by default (#8445)
withastro/astro: packages/integrations/vercel/README.md

/>
```

### useSquooshDev

**Type:** `boolean`<br>
**Available for:** Serverless, Static

> **Added in:** `@astrojs/vercel@3.3.0`

Reviewer: Maybe also add a `**Default**:  false` here?

> When enabled, in development mode a Squoosh-based image service will be used in development instead of Sharp. This is useful if you cannot install Sharp's dependencies on your development machine.

Reviewer: ```suggestion
Allows you to configure a Squoosh-based image service to be used in development instead of Sharp. This is useful if you cannot install Sharp's dependencies on your development machine.
```

/>
```

### devImageService

**Type:** `'sharp' | 'squoosh' | string`<br>
**Available for:** Serverless, Static
**Added in:** `@astrojs/vercel@3.3.0`
**Default**: 'sharp'

> Allows you to configure a which image service to use in development. This can be useful if you cannot install Sharp's dependencies on your development machine and prefer to use Squoosh instead.

Reviewer: ```suggestion
Allows you to configure which image service to use in development. This can be useful if you cannot install Sharp's dependencies on your development machine, but using another image service like Squoosh would allow you to preview images in your dev environment.
```
I know this might sound like a little much, but I wonder whether adding a purpose to this statement (pick the reason you think makes sense!) helps to describe the choice and also really emphasize that this is JUST for development. (Build is unaffected.  <-- and you could even add something like that, if you wanted!)
